deploy/charts/tiflow-operator/templates/manager-rbac.yaml

```diff
@@ -0,0 +1,187 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
if code is updated, do we need to rewrite this rbac file manually?

Reply:
i am sorry but yes, unless `kustomize` support generate charts as well